examples_script/napoleon/napoleon.py

```diff
@@ -1,46 +1,23 @@
 #!/usr/bin/env python3
 
 
-import sys
-sys.path.append("../../.")
-from autocards import Autocards
+# import sys
+# sys.path.append("../../.")
+from autocards.autocards import Autocards
 from pathlib import Path
 
 
 prefix = "On Napoléon : "
-file = Path("./napoleon.txt")
+file = Path("/path/to/Autocards/examples_script/napoleon/napoleon.txt")
 
 if not file.exists():
     print("File not found!")
     raise SystemExit()
 else:
-    full_text = file.read_text()
+    full_text = file.read_text()[:1_000]
```

Comment:
I think it would be best to reduce the size of the example text rather than silently reading only a portion don't you think ? Anyhow why do you feel this is necessary ? Is the text really that big ?

Reply:
I was doing it to get quick results since it is an example script. I think it would be easier for people understand the output. I agree, it would be better to reduce the example text than this since people might not look to closely at it.

Reply:
It's not that big so I'll go ahead and just undo this change here and leave the file alone